<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Basler_QTClass</class>
 <widget class="QMainWindow" name="Basler_QTClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1047</width>
    <height>829</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Basler_QT</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="DiscoverCam">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Discover Cameras</string>
    </property>
   </widget>
   <widget class="QComboBox" name="CameraListComboBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>511</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="OpenSelectedButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Selected</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CloseCamButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Close Camera</string>
    </property>
   </widget>
   <widget class="QLabel" name="CamOpenStatusLabel">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>70</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(170, 255, 255);
border-width: 1px;
border-style: solid;
border-color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="ImageShowWidget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>641</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="SingleShotButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>630</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Single Shot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ContinuousShotButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>630</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Continuous Shot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StopButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>630</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>690</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Exposure Time</string>
    </property>
   </widget>
   <widget class="QSlider" name="ExposureTimeSlider">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>700</y>
      <width>421</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1047</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Basler_QT.qrc"/>
 </resources>
 <connections/>
</ui>
